android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1662,14 +1663,14 @@ static public ChromeTabbedActivity getChromeTabbedActivity() {
         return null;
     }
 
-    static public BraveActivity getBraveActivity() {
+    static public BraveActivity getBraveActivity() throws ActivityNotFoundException {
```

Comment:
should we add @NotNull annotation to this method?

Reply:
updated